src/settings-ui/Settings.UI/SettingsXAML/OOBE/Views/OobeCmdNotFound.xaml

```diff
@@ -0,0 +1,32 @@
+﻿<Page
+    x:Class="Microsoft.PowerToys.Settings.UI.OOBE.Views.OobeCmdNotFound"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:controls="using:Microsoft.PowerToys.Settings.UI.Controls"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:Microsoft.PowerToys.Settings.UI.OOBE.Views"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:toolkitcontrols="using:CommunityToolkit.WinUI.UI.Controls"
+    mc:Ignorable="d">
+
+    <controls:OOBEPageControl x:Uid="Oobe_CmdNotFound" HeroImage="ms-appx:///Assets/Settings/Modules/OOBE/CmdNotFound.png">
+
+        <controls:OOBEPageControl.PageContent>
+            <StackPanel Orientation="Vertical">
+                <TextBlock x:Uid="Oobe_HowToUse" Style="{ThemeResource OobeSubtitleStyle}" />
+
+                <controls:ShortcutWithTextLabelControl x:Name="HotkeyControl" x:Uid="Oobe_CmdNotFound_HowToUse" />
```

Comment:
@stefansjfw @carlos-zamora This should have been a MarkdownTextblock. Who wants to fix it? :)
![image](https://github.com/microsoft/PowerToys/assets/65828559/20dc7d24-7744-4225-93a9-4b403adfc884)

Reply:
Thank you!
Only saw this now :) Ended fixing it up in the workflow improvement PR after noticing it too.